wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,78 @@
+# How to install drivers and fix them 
+Hello, in this post you will know how to download or update your drivers for your computer.
```

Comment:
small grammar corrections

"In this post, you will learn how to download or updating drivers on your computer."

In my opinion, the "hello" in the beginning is not really needed but ig you can leave it there lol.